.github/workflows/main.yml

```diff
@@ -565,7 +474,7 @@ jobs:
   npm-release:
     name: 📦 npm release
     runs-on: ubuntu-latest
-    needs: [js-build-release, python-build-check, python-build-check-armv7, python-sdist, build-release-on-osx, build-release-on-ubuntu]
+    needs: [js-build-release, python-sdist, build-release-on-osx, build-release-on-ubuntu]
```

Comment:
should not depends on python-sdist since it does not run always, but I have to think about this since it the logic is not so obvious in this part